spec/support/test_server.rb

```diff
@@ -34,9 +34,11 @@ def find_available_port
   def call(env)
     @root = File.expand_path(File.dirname(__FILE__))
     path = Rack::Utils.unescape(env['PATH_INFO'])
-    case path
-    when '/get_success'
+    method = env['REQUEST_METHOD']
+    if method == 'GET' && path == '/get_success'
       [ 200, {'Content-Type' => 'text/plain'}, 'success' ]
+    elsif method == 'POST' && path == '/post_success'
+      [ 200, {'Content-Type' => 'text/plain'}, 'post success']
```

Comment:
Space inside square brackets detected.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.